docs/dev/news/2958.fix

```diff
@@ -0,0 +1 @@
+The JS runtime now handles pathological number-to-BigNumber equality comparisons.
```

Comment:
Do we actually want to include this in the release notes?  I think of the "fixed" section as "things users might have run into / might want to know they should try again since the problem is fixed.

Reply:
That's fair, especially since this issue only comes up if writing malformed externs. I'll remove the note.